Tests/Objects/IGTestStoryboardCell.m

```diff
@@ -11,4 +11,10 @@
 
 @implementation IGTestStoryboardCell
 
+- (id)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder:aDecoder];
+    self.label.text = @"";
+    return self;
+}
```

Comment:
Oh lol shoot, been living in code too much. I missed the `@IBOutlet`. Do you mind removing this? 😭